envoy/http/header_map.h

```diff
@@ -449,6 +449,21 @@ class HeaderMap {
    */
   virtual uint64_t byteSize() const PURE;
 
+  // @return uint32_t the max size of the header map in kilobyte.
+  virtual uint32_t maxHeadersKb() const PURE;
+
+  // @return uint32_t the max count of headers in a header map.
```

Comment:
format nit: the comment in this file is /**xxxx */ style

Reply:
Good catch. Done